### DISCUSSION TITLE
Swell and Algolia not syncing

### DISCUSSION THREAD

**Discussion by JoeMatkin** [swellstores/community#466] [Help] [answered]
Hello,

When I try to sync Swell's products to Algolia it doe's not work. On Algolia none of the products from Swell are shown.

I'm sure I'm entering the Application ID and Admin API Key in correctly.

Is this integration still working?

<img width="396" alt="Screenshot 2024-01-03 at 18 00 46" src="https://github.com/swellstores/community/assets/72705313/cc7543e2-2e5a-4d69-86e5-ea415aaaa506">

**Comment by logeshswell** [comment]
I can confirm that the integration is working fine. 
To further investigate your issue, can you share your swell store URL?

**Reply by JoeMatkin** [reply]
It's - [https://x-distributions.swell.store](url)

Thanks

**Reply by logeshswell** [reply, marked as answer]
Thanks for bringing it up, looks like there's an issue with the product sync process.
We'll take a deeper look at this issue and get back to you if I get some updates.

**Reply by JoeMatkin** [reply]
No worries, do you have an estimate as to when this will be resolved, thanks

**Reply by JoeMatkin** [reply]
ShipStation integration appears to be having issue too?


<img width="918" alt="Screenshot 2024-01-05 at 08 31 43" src="https://github.com/swellstores/community/assets/72705313/5468cfe5-4643-4b63-bada-9843ac19c73f">

**Reply by logeshswell** [reply]
We'll take a deeper look at Shipstation integration as well and get back to you if I get some updates.

**Reply by JoeMatkin** [reply]
Hi, any updates? @logeshswell

**Reply by logeshswell** [reply]
No updates as of now, but I will keep you posted if there's an update.

**Comment by logeshswell** [comment]
Algolia and Shipstation sync is fixed at our end. Can you check and confirm, please?

**Reply by JoeMatkin** [reply]
Great!

Algolia appears to be working, and so does ShipStation.

Thanks
